pkg/ble/Makefile

```diff
@@ -0,0 +1,12 @@
+SHELL = /bin/bash
+
+build:
+	@GOOS=linux CGO_ENABLED=0 GOARCH=arm go build -o ble-go
```

Comment:
The arm platform should be notified to the user.

Reply:
/lgtm, it had been fixed